### PR TITLE
Update `Project`s to run examples

### PR DESCRIPTION
**Pull request by simone-silvestri**
Examples required some updates to the `Project.toml`s to run

### CODE REVIEW

**Review by glwagner — APPROVED** [review]

**Comment by glwagner** [comment]
What about the status of #156

**Comment by navidcy** [comment]
> What about the status of #156

I've been included this updates there.
But the examples need 20hrs to run and they keep failing and it's been impossible to figure out... It has worn me out. From time to time I return to it but this 20hrs wait makes it impossible to work because I make a tweak and then need to wait for a day, I forget things etc... I'll try to get back to it!

File: src/DataWrangling/JRA55.jl [hunk 1]
```diff
@@ -444,7 +444,7 @@ function JRA55_field_time_series(variable_name;
     close(ds)
 
     N = (Nrx, Nry)
-    H = min.(N, (3, 3))
+    H = min.(N, (7, 7))
```
Comment (glwagner) [inline src/DataWrangling/JRA55.jl:447]:
```suggestion
    H = map(min, (N, N), (7, 7))
```

try not to use broadcasting unless you need it

**Comment by simone-silvestri** [comment]
Superseded by #156

**Pull request closed without merging**